src/common/src/config.rs

```diff
@@ -1518,7 +1518,7 @@ pub fn extract_storage_memory_config(s: &RwConfig) -> StorageMemoryConfig {
         .storage
         .high_priority_ratio_in_percent
         .unwrap_or(default::storage::high_priority_ratio_in_percent());
-    let large_query_memory_usage_mb = s
+    let prefetch_buffer_capacity_mb = s
         .storage
         .shared_buffer_capacity_mb
         .unwrap_or((100 - high_priority_ratio_in_percent) * block_cache_capacity_mb / 100);
```

Comment:
No. We just to set a value like block-cache.
As the performance benchmark, it can not be too small because there may be several iterator using prefetch.
For example: if there are 256 vnode reading hummock and each vnode creates 8 iterator using prefetch, they will cost 256 * 8 * 16 * 64KB = 2GB memory.
So it can not be a small number.